### DISCUSSION TITLE
Bremsstrahlung Splitting

### DISCUSSION THREAD

**Discussion by piliop1** [nrc-cnrc/EGSnrc#1015] [Questions] [answered]
I use EGSnrc for KV energies (125KV).
When I use ‘UBS’ and ‘None’ Bremsstrahlung splitting, I can see in the report (.egslst file)  that Bremsstrahlung events are present.
Using beamdp to list the particles in the phase space file I can see that we do have bremsstrahlung photons because LATCH BIT 0 is set to 1 for those photons.
Whenever I set Directional Bremsstrahlung splitting, I notice in the report (.egslst file) that the number of Bremsstrahlung events is ZERO!!! (I tried a plenty of parameter combinations for splitting field radius, SSD and Bremsstrahlung Splitting Number but no success).
Using again beamdp to list the generated particles I can see that Bremsstrahlung photons are present because of LATCH’s BIT0 set to 1.
Can please somebody explain this behavior? How is it possible to get Bremsstrahlung photons without Bremsstrahlung events?
Thank you!

**Comment by rtownson** [comment, marked as answer]
This was a bug! Thanks for reporting it. Fixed in https://github.com/nrc-cnrc/EGSnrc/pull/1019

Note that the brems events were occurring, they just weren't getting added to that counter.

**Reply by piliop1** [reply]
Thank you very much for your answer!! This issue has bothered me for a long time!
